src/site/_filters/paged-navigation.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Takes an object and determines how many links to other pages in the same paginated collection to display.
+ * @param {object} paged Page details including the `index` in the pagination, how many `pages` there are, and the `href` of the page.
+ * @return {Array<object>} An array of up to 8 items to display, including href and index.
+ */
+module.exports = function pagedNavigation(paged) {
+  const shiftBy = 4;
+  const start = paged.index - shiftBy > 0 ? paged.index - shiftBy : 0;
```

Comment:
Can you write this as `Math.max(paged.index - shiftBy, 0)`, and similarly using `Math.min` below? It might be easier to read.